renderer/src/pages/wallet/TransferWrapper.tsx

```diff
@@ -0,0 +1,73 @@
+import Text from 'src/components/Text'
+import { Wallet } from 'src/hooks/StationWallet'
+import DestinationAddressForm from './DestinationAddressForm'
+import BalanceControl from './BalanceControl'
+import EditDestinationAddressForm from './EditDestinationAddressForm'
+import { useState } from 'react'
+import ConfirmTransfer from './ConfirmTransfer'
+
+const SEND_THRESHOLD = 0.01
+
+const TransferWrapper = ({
+  destinationFilAddress,
+  stationAddress,
+  processingTransaction,
+  walletBalance,
+  editDestinationAddress,
+  transferAllFundsToDestinationWallet,
+  dismissCurrentTransaction
+}: {
+  walletBalance: Wallet['walletBalance'];
+  destinationFilAddress?: Wallet['destinationFilAddress'];
+  stationAddress?: Wallet['stationAddress'];
+  processingTransaction : Wallet['processingTransaction'];
+  editDestinationAddress:Wallet['editDestinationAddress'];
+  transferAllFundsToDestinationWallet: Wallet['transferAllFundsToDestinationWallet'];
+  dismissCurrentTransaction: Wallet['dismissCurrentTransaction'];
+}) => {
+  const [showConfirm, setShowConfirm] = useState(false)
```

Comment:
```suggestion
  const [isShowingConfirm, setIsShowingConfirm] = useState(false)
```

`showConfirm` sounds like a function name to me